corehq/apps/hqmedia/views.py

```diff
@@ -622,9 +634,9 @@ def _files(files):
                 # TODO: make RemoteApp.create_all_files not return media files
                 extension = os.path.splitext(name)[1]
                 data = _encode_if_unicode(f) if extension in text_extensions else f
-                yield (get_name(name), data)
+                yield (_get_name(name), data)
```

Comment:
Since `_get_name(name)` is removing the build profile path element from `name`, it appears that line 633 (`if name not in skip_files:`) will never evaluate to false (no files will ever be skipped) when a build profile is given (assuming all values of `name` will start with `build_profile_id + '/'`). Is that correct/intentional?